tests/SBRPTests/SBRPTests.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace SBRPTests;
+
+public class ReferencePackagesTest : IClassFixture<SBRPFixture>, IDisposable
```

Comment:
This name doesn't accurately reflect what the test class does.  It is not validating the reference packages themselves rather it is testing the generator.  Additionally it looks like the current test is validating both reference and text only packages.

Reply:
I will update it to GenerateScriptRegressionTests